src/anemoi/training/config/diagnostics/eval_rollout.yaml

```diff
@@ -24,6 +24,7 @@ plot:
   #Defining the accumulation levels for precipitation related fields and the colormap
   accumulation_levels_plot: [0, 0.05, 0.1, 0.25, 0.5, 1, 1.5, 2, 3, 4, 5, 6, 7, 100] # in mm
   cmap_accumulation: ["#ffffff", "#04e9e7", "#019ff4", "#0300f4", "#02fd02", "#01c501", "#008e00", "#fdf802", "#e5bc00", "#fd9500", "#fd0000", "#d40000", "#bc0000", "#f800fd"]
+  precip_like_vars: {"tp", "cp"}
```

Comment:
I am wondering about the naming of this. Is the cmap/features associated to just precip and other associated variables or this due to the bounding/treatment with do on precip?

Reply:
Yes good point, I was thinking we might be able to find a better name. This plotting will only ever really need to be applied to precipitation type variables with units of kg m-2 s-1 I think, so runoff (ro) and snowfall (sf) etc. Not sure if we could go for something like precipitation_related_fields, or moisture_variables, or just leave as precip_like_vars. Do you have any other ideas you think might be clearer?